scripts/claude_start.sh

```
@@ -0,0 +1,1734 @@
+            echo -e "${BLUE}🤖 Qwen mode selected - using Qwen3-Coder via vast.ai${NC}"
+            echo -e "${BLUE}💡 Features: Remote Qwen3-Coder model via vast.ai GPU instances${NC}"
+            echo -e "${BLUE}💡 Model: qwen3-coder (optimized for coding tasks)${NC}"
+            echo ""
+
+            # Check if API proxy is available
+            # TERMINAL SESSION PRESERVATION: Never exit terminal on errors - let users Ctrl+C to go back
+            API_PROXY_PATH="$HOME/projects/claude_llm_proxy/api_proxy.py"
+            if [ ! -f "$API_PROXY_PATH" ]; then
+                echo -e "${RED}❌ LLM self-hosting repository not found${NC}"
+                echo -e "${BLUE}💡 LLM proxy (for Qwen and Cerebras) is maintained in the separate claude_llm_proxy repository${NC}"
+                echo -e "${BLUE}💡 Clone it with: cd ~/projects && git clone $(git config --get remote.origin.url)
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)